shared/settings/account/index.tsx

```diff
@@ -241,8 +246,8 @@ const styles = Styles.styleSheetCreate(() => ({
     height: 16,
     width: 16,
   },
-  section: Styles.platformStyles({
```

Comment:
can't you just encode this in one platformStyles?

Reply:
Hm, wonder how best to do that without duplicating. We want one style for phone and one for not-phone, with a different maxWidth for tablet and desktop. But we only have common/isElectron/isPhone/isMobile/isTablet to work with.

Reply:
why is it not
```
Styles.platformStyles({
  common: {
...Styles.padding(
        Styles.globalMargins.small,
        Styles.globalMargins.mediumLarge,
        Styles.globalMargins.medium,
        Styles.globalMargins.small
      ),
 } ,
isElectron : {
    maxWidth: 600,
},
isTablet: {
      maxWidth: Styles.globalStyles.largeWidthPercent,
    },
isPhone: {
   ...Styles.padding(Styles.globalMargins.small, Styles.globalMargins.small, Styles.globalMargins.medium),
}
})
```
?

Reply:
Oh right -- I was turned off by the mixing of spreading the `Styles.padding()` returns from `isPhone` on top of `common`, with a different number of args in each case. Whether the spread does the right thing (e.g. re-uses `common`'s `left` without recalculating it) depends on the implementation of `Styles.padding()` and that was unsatisfying.

But if I look at the implementation of `Styles.padding()` I can convince myself that it behaves okay, so I'm good with switching to your ver if you prefer it, lemme know.

Reply:
i think that function it tiny and does what you expect. anyways either way no biggie.